README.rst

```diff
@@ -98,14 +98,35 @@ This will add these paths to Django:
 * ``/oauth2/callback`` where ADFS redirects back to after login. So make sure you set the redirect URI on ADFS to this.
 * ``/oauth2/logout`` which logs out the user from both Django and ADFS.
 
-You can use them like this in your django templates:
+```suggestion
```

Comment:
Will this render properly?

Reply:
Ack, no. That is a github feature. I'll clean this up